test/python/pulse/test_instructions.py

```diff
@@ -267,15 +267,14 @@ def test_parameterized_call(self):
         self.assertTrue(call.is_parameterized())
         self.assertEqual(len(call.parameters), 2)
 
-    def test_assign_parameters(self):
-        """Test assigning parameter doesn't immediately update program."""
-        call = instructions.Call(subroutine=self.function)
-        call.assign_parameters({self.param1: 0.1, self.param2: 0.2})
-
-        self.assertFalse(call.is_parameterized())
+    def test_call_initialize_with_parameter(self):
+        """Test call instruction with parameterized subroutine with initial dict."""
+        init_dict = {self.param1: 0.1, self.param2: 0.5}
+        call = instructions.Call(subroutine=self.function, value_dict=init_dict)
```

Comment:
Is this the only way to assign parameters, now? How would I build a `Schedule` (directly) and then assign parameters?

Reply:
No. You can call subroutine without parameters and assign them to the created schedule. This allows user to reuse subroutine within the same schedule with different parameters.

Reply:
I added new test for this.